protocol/synthetix/contracts/modules/core/MarketManagerModule.sol

```diff
@@ -237,7 +241,10 @@ contract MarketManagerModule is IMarketManagerModule {
         if (msg.sender != market.marketAddress) revert AccessError.Unauthorized(msg.sender);
 
         // min delegate time should not be unreasonably long
-        uint maxMinDelegateTime = Config.readUint(_CONFIG_SET_MARKET_MIN_DELEGATE_MAX, 86400 * 30);
+        uint256 maxMinDelegateTime = Config.readUint(
+            _CONFIG_SET_MARKET_MIN_DELEGATE_MAX,
+            86400 * 30
```

Comment:
save this as a named constant instead?

Reply:
yea I need to figure out a better wya to do this in general, not just this isntance. tbd.